Service/src/routes/charts.py

```diff
@@ -96,8 +106,13 @@ def formatDataInterDay(jsonData):
 
 
 def formatDataIntraday(jsonData):
-   data = [] 
+   data = []
+   NoneItemsIEX = 0 
    for item in jsonData:
+      if item['average'] is None:
+         print(item)
```

Comment:
no print statement --> blocker for merge

use the logger object even when you're debugging

Reply:
Removed